aws_lambda_powertools/event_handler/openapi/models.py

```diff
@@ -19,12 +18,7 @@ class Contact(BaseModel):
     url: Optional[AnyUrl] = None
     email: Optional[str] = None
 
-    if PYDANTIC_V2:
-        model_config = {"extra": "allow"}
-    else:
-
-        class Config:
-            extra = "allow"
+    model_config = {"extra": "allow"}
```

Comment:
can we perhaps define it as a top-level constant to reduce memory usage?

It looks like we always allow extra keys for these models

Reply:
Yeah, make sense! Done.